src/xcode/ENA/ENA/Source/Scenes/Events/Store/Model/TraceLocation.swift

```diff
@@ -3,9 +3,82 @@
 //
 
 enum TraceLocationType: Int {
-	case type1
-	case type2
-	case type3
+	case locationTypeUnspecified // = 0
```

Comment:
Was just wondering, if we add it as a comment anyway (which I think makes sense!), we could also just assign it directly 🤔

```suggestion
	case locationTypeUnspecified = 0
```

Reply:
I will add them, they are not needed but if things change it might help